src/components/calcite-popover/calcite-popover.stories.ts

```diff
@@ -56,70 +63,34 @@ export default {
   }
 };
 
-export const Simple = (): string => {
-  return `
-      <div>
-        ${referenceElementHTML}
-        <calcite-popover
-        class="calcite-theme-light"
-          ${boolean("dismissible", false)}
-          ${boolean("disable-flip", false)}
-          ${boolean("disable-pointer", false)}
-          reference-element="reference-element"
-          placement="${select("placement", calcite_placements, "auto")}"
-          offset-distance="${number("offset-distance", 6)}"
-          offset-skidding="${number("offset-skidding", 0)}"
-          ${boolean("open", true)}
-          text-close="${text("text-close", "Close")}"
-        >
-          ${contentHTML}
-        </calcite-popover>
-      </div>
-    `;
-};
-
-export const RTL = (): string => {
-  const { theme } = ATTRIBUTES;
-
-  return `
-      <div dir="rtl">
-        ${referenceElementHTML}
-        <calcite-popover
-          class="${select("class", theme.values, theme.defaultValue)}"
-          ${boolean("dismissible", false)}
-          ${boolean("disable-flip", false)}
-          ${boolean("disable-pointer", false)}
-          reference-element="reference-element"
-          placement="${select("placement", calcite_placements, "auto")}"
-          offset-distance="${number("offset-distance", 6)}"
-          offset-skidding="${number("offset-skidding", 0)}"
-          ${boolean("open", true)}
-          text-close="${text("text-close", "Close")}"
-        >
-          ${contentHTML}
-        </calcite-popover>
-      </div>
-    `;
-};
-
-export const DarkMode = (): string => {
-  return `
-      <div>
-        ${referenceElementHTML}
-        <calcite-popover
-        class="calcite-theme-dark"
-          ${boolean("dismissible", false)}
-          ${boolean("disable-flip", false)}
-          ${boolean("disable-pointer", false)}
-          reference-element="reference-element"
-          placement="${select("placement", calcite_placements, "auto")}"
-          offset-distance="${number("offset-distance", 6)}"
-          offset-skidding="${number("offset-skidding", 0)}"
-          ${boolean("open", true)}
-          text-close="${text("text-close", "Close")}"
-        >
-          ${contentHTML}
-        </calcite-popover>
-      </div>
-    `;
-};
+export const Simple = stepStory(
+  (): string => html`
+    <div style="width: 400px;">
+      ${referenceElementHTML}
+      <calcite-popover
+        ${boolean("dismissible", false)}
+        ${boolean("disable-flip", false)}
+        ${boolean("disable-pointer", false)}
+        reference-element="reference-element"
+        placement="${select("placement", calcite_placements, "auto")}"
+        offset-distance="${number("offset-distance", 6)}"
+        offset-skidding="${number("offset-skidding", 0)}"
+        ${boolean("open", true)}
+        text-close="${text("text-close", "Close")}"
+      >
+        ${contentHTML}
+      </calcite-popover>
+    </div>
+  `,
+  new Steps()
+    .wait(calciteHydratedSelector)
```

Comment:
☝️ I'm wondering if this step can be included in the `stepStory` decorator since we'll want to wait for the hydrated class to appear before doing any other step in all components?

Reply:
I'm not sure we can add it to the front there.